ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -129,6 +129,13 @@ public final class AliasTokens: NSObject {
         case brandStroke1
         case brandStroke1Pressed
         case brandStroke1Selected
+        case brandStrokeTint
```

Comment:
Looks like the other `brandStroke` tokens are in `ColorProviding`, so this should also be added. Which I think will be breaking.